Makefile

```diff
@@ -38,7 +43,7 @@ clean: checks
 running: options.env
 	oc process -f templates/running-job.yaml --param-file options.env  | oc apply -f -
 
-hibernating: options.env
+hibernate: options.env
```

Comment:
![bitmoji](https://sdk.bitmoji.com/render/panel/8d70afe2-8030-4735-b895-076a9a91f1eb-d47740f2-2bd8-46cf-956f-b4256c3fd3e8-v1.png?transparent=1&palette=1&width=246)

Reply:
I changed this to hibernate because the doc had it as hibernate. I thought I pushed a change to make them both hibernating to match what Hive did. I guess not.

Reply:
This was fixed.